src/control/server/storage/bdev/backend.go

```diff
@@ -42,6 +46,7 @@ type (
 		controllers []spdk.Controller
 
 		initialized bool
+		vmdEnabled  bool
```

Comment:
Rather than making this a property of the backend object, I think this should instead be passed in as a parameter to `init()`. This will remove the need for getter/setter methods and will make it easier to reason about how the code works.

Reply:
how would you persist the enable? you would need to pass parameter also to scan(), keeping this as a backend property means it can be sent once and the persistence between local and remote provider instances can be maintained with forwarding Logic.